src/components/Card.astro

```diff
@@ -0,0 +1,31 @@
+---
+// Properties
+type Props = {
+  description: string;
+  imgAlt: string;
+  imgSrc: string;
+  link: string;
+  title: string;
+  variant?: "default" | "padded";
+};
+const {
+  description,
+  imgAlt,
+  imgSrc,
+  link,
+  title,
+  variant = "default",
```

Comment:
Make the default "", rather than a magic word.

Process what they pass in the code, rather than inline in the HTML.

With Astro, it's best to keep the HTML part as plain as possible, so try to do any logic before you get there if you can.